src/messages/lang.json

```diff
@@ -0,0 +1,3 @@
+{
+	"langInstructions": "Welcome!<br /><br />This tool will make using a computer easier.<br /><br />If you need help at any time, press the H key. To continue, select the arrow below."
```

Comment:
can you change "... select the arrow below." to "... select start." Also is it possible to use `<p>` instead of the `<br />` tags.